### PR TITLE
[ci] run coverage for adapter tests

### PR DESCRIPTION
**Pull request by pbalcer**

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -225,6 +230,19 @@ jobs:
...
+    - name: Quick Coverage Info
+      if: matrix.compiler.c == 'gcc'
+      working-directory: ${{github.workspace}}/build
+      run: ctest -T Coverage
+
+    - name: Upload coverage to Codecov
+      if: matrix.compiler.c == 'gcc'
+      continue-on-error: true
+      uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
+      with:
```
Comment (lukaszstolarczuk) [inline .github/workflows/cmake.yml:242]: we could add `flags: adapters` to distinguish this from other coverage report(s)

**Review by lukaszstolarczuk — COMMENTED** [review]
is this PR still valid? if so, please rebase

**Comment by pbalcer** [comment]
> is this PR still valid? if so, please rebase

it doesn't work because I can't get codecov to upload through the proxy:
```
Run codecov/codecov-action@eaaf4bedf32dbdc[6](https://github.com/oneapi-src/unified-runtime/actions/runs/7490304252/job/20388853697?pr=1165#step:11:7)b[7](https://github.com/oneapi-src/unified-runtime/actions/runs/7490304252/job/20388853697?pr=1165#step:11:8)20b63067d99c4d77d6047d
==> linux OS detected
Error: connect ETIMEDOUT 104.22.23.101:443
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16)
```
Not really sure what's the issue. Didn't have much time to debug this.

**Comment by lukaszstolarczuk** [comment]
> > is this PR still valid? if so, please rebase
> 
> it doesn't work because I can't get codecov to upload through the proxy:
> 
> ```
> Run codecov/codecov-action@eaaf4bedf32dbdc[6](https://github.com/oneapi-src/unified-runtime/actions/runs/7490304252/job/20388853697?pr=1165#step:11:7)b[7](https://github.com/oneapi-src/unified-runtime/actions/runs/7490304252/job/20388853697?pr=1165#step:11:8)20b63067d99c4d77d6047d
> ==> linux OS detected
> Error: connect ETIMEDOUT 104.22.23.101:443
>     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16)
> ```
> 
> Not really sure what's the issue. Didn't have much time to debug this.

ahh, I see. I quickly looked into it and I don't see the proxy param in the codecov action. There's an issue, I guess, confirming that: https://github.com/codecov/codecov-action/issues/700#issuecomment-1135401880  - leaving this for future ref.

**Comment by pbalcer** [comment]
They added proxy to the action in the version that this PR is using: https://github.com/codecov/codecov-action/tree/v3.1.4

But it doesn't seem to work. There's now v4 of this action, maybe it's fixed?

**Pull request closed without merging**